db/migrate/20211130190839_fix_webster_grove_coordiantes.rb

```diff
@@ -0,0 +1,62 @@
+class FixWebsterGroveCoordiantes < ActiveRecord::Migration[6.0]
```

Comment:
It is not best practice to run data migrations this way. These migrations are for database structure/schema changes only, not database records. Please move this to a rake task and run it in the environments where you need to change this data. See here for more info [Data Migrations](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations.689832034.html#vets-apidatabasemigrations-DataMigrations)

Reply:
I moved the migration to a rake task